src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="container">
+      <div class="calendar">
```

Comment:
```suggestion
      <div class="calendar calendar__start-day--mon">
```
У календаря должен быть модификатор который по сути считает отступ влево, чтобы если ты изменил модификатор на wed то календарь начинался с 3 столбика как в настоящем календаре со среды